webpack.config.js

```diff
@@ -0,0 +1,37 @@
+const {resolve} = require('path');
+
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const HtmlWebpackPluginConfig = new HtmlWebpackPlugin({
+  template: './index.html',
+  filename: 'index.html',
+  inject: 'body'
+})
+
+module.exports = env => {
+  env = env || { dev: true }
+
+  return {
+    context: resolve('client'),
+    entry: './index.js',
+    output: {
+      path: resolve('dist'),
+      filename: 'bundle.js',
+      publicPath: "/assets/",
+      pathinfo: !env.prod
+    },
+    module: {
+      rules: [
+        { test: /\.(js|jsx)$/, loader: 'babel-loader' },
+        { test: /\.css$/, use: [ 'style-loader', 'css-loader' ] }
+      ]
+    },
+    plugins: [HtmlWebpackPluginConfig],
+    devServer: {
```

Comment:
I think the we should use dev server on dev and bundle everything in the deploy procedure. The other options is to use webpack watch mode instead of the dev server.